src/controllers/albumController.js

```diff
@@ -51,6 +107,20 @@ const updatePriceOfAlbums = async (req, res) => {
   logger.info(`${bulkWriteResponse.modifiedCount} Albums updated successfully`);
   const updatedAlbums = await Album.find({ _id: { $in: ids } }); // Fetches the updated albums from the database using the _id values in the ids array. //see Implicit $in in mongoose docs
   res.status(StatusCodes.OK).json({ albums: updatedAlbums });
+  /*
+     #swagger.summary = 'Update prices of albums passed in req.body'
+	#swagger.parameters['albums to update prices'] = {
```

Comment:
swagger-autogen is getting tripped up by the `req.body.map` in this endpoint so it thinks there's some parameter called `"map(update..."`. we can tell it to ignore that and just go with what you've explicitly defined here:

```suggestion
	#swagger.autoBody = false
	#swagger.parameters['albums to update prices'] = {
```